### DISCUSSION TITLE
How about we change the name of SUMMARY.md to README.md

### DISCUSSION THREAD

**Discussion by Abhiroop25902** [codeiiest-dev/Algorithms#46] [Ideas]
Thinking about this as README.md files are auto shown in the GitHub page, this could make navigation faster within the GitHub Repo in web......

Here is a sample of what I trying to convey....

![Algorithms_src_MachineLearningAlgorithms at main · codeiiest-dev_Algorithms - Google Chrome 19-03-2021 11_47_09 AM (2)](https://user-images.githubusercontent.com/69428539/111739464-18b07480-88a9-11eb-9ee1-a6d147200179.png)

**Comment by arnabsen1729** [comment]
I understand your point, but we are using mdbooks for the static generation of the website, and SUMMARY.md is the left side table of contents, You can read more about it in their documentation.

![image](https://user-images.githubusercontent.com/51032928/111739876-c6238800-88a9-11eb-9b0e-909500ab6d7e.png)
This is how the SUMMARY.md is rendered.

**Reply by Abhiroop25902** [reply]
Ok, Understood 😅
